mixins/demo/localize-mixin-mission.js

```diff
@@ -22,8 +25,16 @@ class Mission extends LocalizeDynamicMixin(LitElement) {
 			link1: linkGenerator({ href: 'https://wikipedia.org/wiki/Culture_change', target: '_blank' }),
 			link2: chunks => markup`<d2l-link href="https://wikipedia.org/wiki/Earth" target="_blank"><em>${chunks}</em> ${surnameMarkup}</d2l-link>`
 		};
+		let a;
```

Comment:
`const a = undefined`? I've got this thing against `let` statements where they're not absolutely needed 😆.

Reply:
Uh oh. I agree, but I've also got a thing against literal `undefined` where they aren't absolutely needed. I think my brain is doing a kernel panic.